starter-code/scripts/article.js

```diff
@@ -50,13 +50,30 @@ Article.fetchAll = function() {
     // When rawData is already in localStorage,
     // we can load it with the .loadAll function above,
     // and then render the index page (using the proper method on the articleView object).
-    Article.loadAll(); //TODO: What do we pass in to loadAll()?
+    Article.loadAll(JSON.parse(localStorage.getItem('data'))); //TODO: What do we pass in to loadAll()?
```

Comment:
Thx! Yes, it is the reason that the data file cannot be loaded. I revised it and it is working now!